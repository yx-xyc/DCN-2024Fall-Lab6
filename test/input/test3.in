PASSIVE
SYN
ACK
SDATA
RDATA
INVALID_EVENT
FIN
CLOSE
ACK