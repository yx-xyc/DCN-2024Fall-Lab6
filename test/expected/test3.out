Event PASSIVE received, current State is CLOSED
Event SYN received, current State is LISTEN
Event ACK received, current State is SYN_RCVD
DATA sent 1
DATA received 1
Error: unexpected Event: INVALID_EVENT
Event FIN received, current State is ESTABLISHED
Event CLOSE received, current State is CLOSE_WAIT
Event ACK received, current State is LAST_ACK
